libs/designsystem/src/lib/components/page-local-navigation/page-local-navigation.component.html

```diff
@@ -18,17 +18,17 @@
             [attr.aria-label]="badge.description"
             [themeColor]="badge.themeColor"
           >
-            <ng-container *ngIf="badge.content.name; else badgeTextContent">
+            <ng-container *ngIf="$any(badge).content.name; else badgeTextContent">
```

Comment:
When there is a union type checking in the template it throws the classic TS error, when trying to access the property of a union type:

```
Property 'name' does not exist on type 'BadgeText | BadgeIcon'.
  Property 'name' does not exist on type 'BadgeText'.
```
I couldn't find a way to use *`ngIf` to do some checks, that would allow TS to infer the type correctly. So I ended up type casting the `badge` as any. If the person reviewing this has any suggestions on how to do it differently, I would love to hear them! :)